### PR TITLE
chore(rxStatus): decompose monolithic source

### PR DESCRIPTION
**Pull request by CITguy**
JIRA: https://jira.rax.io/browse/FRMW-364

### LGTMS
- [x] Dev LGTM
- [x] Dev LGTM

### CODE REVIEW

File: src/rxStatus/scripts/Status.js [hunk 1]
```diff
@@ -1,38 +1,4 @@
...
- *
- * @example
- * <pre>
- * $rootScope.$on('$routeChangeSuccess', function () {
- *     Status.setScope(); // no input results in $rootScope being used
- *     Status.setUtil($rootScope); // forcibly set $rootScope as the scope to be used
- * });
- * </pre>
- */
-.service('StatusUtil', function ($route, $rootScope, Status) {
```
Comment (CITguy) [inline src/rxStatus/scripts/Status.js]: moved to `src/rxStatus/scripts/StatusUtil.js`

File: src/rxStatus/scripts/Status.js [hunk 2]
```diff
@@ -378,29 +344,4 @@ angular.module('encore.ui.rxStatus', ['encore.ui.rxNotify'])
...
     return status;
-})
-/**
- * @ngdoc service
- * @name rxStatus.service:ErrorFormatter
- * @description
- * Provides a helper method to parse error objects for 'message' and format them
- * as necessary for Status.setError()
- */
-.factory('ErrorFormatter', function () {
```
Comment (CITguy) [inline src/rxStatus/scripts/Status.js]: moved to `src/rxStatus/scripts/ErrorFormatter.js`

File: src/rxStatus/scripts/Status.spec.js [hunk 3]
```diff
@@ -97,75 +97,3 @@ describe('rxStatus: Status', function () {
...
-        expect(status.setScope.args[0][0]).to.deep.equal(rootScope);
-    });
-
-    it('StatusUtil: should allow storing of provided scope reference', function () {
-        util.setupScope({ fake: 'object' });
-        expect(status.setScope.args[0][0]).to.deep.equal({ fake: 'object' });
-    });
-});
-
-describe('rxStatus: ErrorFormatter', function () {
```
Comment (CITguy) [inline src/rxStatus/scripts/Status.spec.js]: moved to `src/rxStatus/scripts/ErrorFormatter.spec.js`

File: src/rxStatus/scripts/Status.spec.js [hunk 4]
```diff
@@ -97,75 +97,3 @@ describe('rxStatus: Status', function () {
         });
     });
 });
-
-describe('rxStatus: StatusUtil', function () {
```
Comment (CITguy) [inline src/rxStatus/scripts/Status.spec.js]: moved to `src/rxStatus/scripts/StatusUtil.spec.js`

**Comment by bry4n** [comment]
LGTM

**Comment by 100stacks** [comment]
LGTM

**Pull request merged**